client.go

```diff
@@ -298,9 +302,24 @@ func (c *context) interfaces() []Interface {
 // Packets will be dropped if the buffer is full. Used by NewClient.
 var MaxQueueBuffer = 100
 
+func newTransport() Transport {
+	t := &HTTPTransport{}
+	rootCAs, err := gocertifi.CACerts()
+	if err != nil {
```

Comment:
`rootCAs` isn't a slice for one. It's a `*x509.CertPool` for one. And this certificate bundle is hardcoded in https://github.com/certifi/gocertifi/blob/master/certifi.go